docs/optimization_guide/nncf/ptq/code/ptq_torch.py

```diff
@@ -19,3 +19,18 @@ def transform_fn(data_item):
 
 quantized_model = nncf.quantize(model, calibration_dataset)
 #! [quantization]
+
+#! [inference]
+import openvino.runtime as ov
+from openvino.tools.mo import convert_model
+
+input_fp32 = ... # FP32 model input
+
+# convert PyTorch model to OpenVINO model
+ov_quantized_model = convert_model(quantized_model, example_input=input_fp32)
```

Comment:
```suggestion
input_fp32 = ... # FP32 model input

# export PyTorch model to ONNX model
onnx_model_path = "model.onnx"
torch.onnx.export(quantized_model, input_fp32, onnx_model_path)

# convert ONNX model to OpenVINO model
ov_quantized_model = convert_model(onnx_model_path)
```

Updated the script according the changes in the OV 2023.0.0 cc' @slyalin @yury-gorbachev